### ISSUE TITLE
Where is Building Setting folder from Tutorial

### ISSUE THREAD

**Issue by ankofl** [Unity-Technologies/ECS-Network-Racing-Sample#9]
![Image](https://github.com/user-attachments/assets/9395c41b-70f3-4c18-b242-e3cad50640b5)
![Image](https://github.com/user-attachments/assets/267d87a0-6da0-4773-bc10-c6571c2bb378)
![Image](https://github.com/user-attachments/assets/a46b95c6-eb7f-4a7b-bacd-7c75859963dd)

**Comment by ankofl** [comment]
No, seriously, are you specifying a folder in the tutorial that is not in the project?
![Image](https://github.com/user-attachments/assets/f771109f-f1bc-472c-8cc5-9473fca28f9e)
![Image](https://github.com/user-attachments/assets/f4c541b2-2c71-4277-97be-305d9c261b2e)

**Comment by ankofl** [comment]
Hi @julianunity !

**Comment by ankofl** [comment]
@julianunity 
No, seriously, I thought it was deleted in one of the commits, and the launch information remained in the tutorial, but I looked through several commits and didn't find this folder there either. Where can I find her?

**Comment by sanliuk** [comment]
man really whhere is it

**Comment by sanliuk** [comment]
build setting folder is not there.

**Comment by ankofl** [comment]
Hi @sanliuk 
> man really whhere is it

I have no idea, but at the moment it's better to use [this](https://docs.unity3d.com/Packages/com.unity.template.multiplayer-netcode-for-entities@1.0/manual/index.html) template.

**Comment by jmeyer1980** [comment]
Really? No answer yet on this? I've successfully been able to build both the server side and client side but I cannot get them to connect. Server is consistently not found and I am unsure of how to fix it as... no build settings folder as the tutorial leads us to believe.

**Comment by ankofl** [comment]
> Really? No answer yet on this? I've successfully been able to build both the server side and client side but I cannot get them to connect. Server is consistently not found and I am unsure of how to fix it as... no build settings folder as the tutorial leads us to believe.

Apparently, this is no longer an up-to-date repository and they are focused on [Competitive Action Multiplayer Template](https://discussions.unity.com/t/competitive-action-multiplayer-template-now-released-netcode-for-entities/1573137)

**Comment by julianunity** [comment]
Hello @ankofl @jmeyer1980 @sanliuk.

My appoligies for the really slow response, I am working on updating this Project and the documentation, I will post the updates as soon as possible.

**Comment by julianunity** [comment]
Hello @ankofl @jmeyer1980 @sanliuk.

Please check the new commit with the updated information about the documentation an tutorials of How to Play and build.

**Comment by ankofl** [comment]
Hi @julianunity 

> Hello [@ankofl](https://github.com/ankofl) [@jmeyer1980](https://github.com/jmeyer1980) [@sanliuk](https://github.com/sanliuk).
> 
> Please check the new commit with the updated information about the documentation an tutorials of How to Play and build.

Of course, thank you for correcting the template at least six months later, but I've been using a different template for a long time

**Comment by julianunity** [comment]
Hi @ankofl my appoligies for the long delayed response.

I will close this for now, but I will be attentive to the future entries.

**Comment by ankofl** [comment]
Hi @julianunity 
I think it's worth clarifying one question: do you receive or have you ever received money for working on this repository from Unity? 

If not, then I apologize profoundly for my tactlessness and toxicity! In that case, it was very bad of me, and thank you for your hard work!
